core/src/com/unciv/ui/newgamescreen/NationTable.kt

```diff
@@ -8,53 +8,86 @@ import com.badlogic.gdx.utils.Align
             leaderDisplayLabel.wrap = true
             titleTable.add(leaderDisplayLabel).width(leaderDisplayNameMaxWidth)
-        } else titleTable.add(leaderDisplayLabel)
-        innerTable.add(titleTable).row()
+        } else {
+            titleTable.add(leaderDisplayLabel).align(Align.center).pad(10f,0f)
+        }
+
+        innerTable.add(titleTable).growX().fillY().row()
 
         if (ruleset != null) {
-            val nationUniqueLabel = nation.getUniqueString(ruleset).toLabel(nation.getInnerColor())
-            nationUniqueLabel.wrap = true
-            innerTable.add(nationUniqueLabel).width(internalWidth)
+            titleTable.padBottom(borderWidth) // visual centering including upper border
+            innerTable.background = ImageGetter.getBackground(textBackgroundColor)
+            val lines = nation.getCivilopediaTextLines(ruleset)
+                .filter { it.header != 3 }
+            innerTable.add(MarkupRenderer.render(lines, internalWidth, iconDisplay = IconDisplay.NoLink)).pad(10f)
+            val borderTable = Table()
+            borderTable.background = ImageGetter.getBackground(outerColor)
+            borderTable.add(innerTable).pad(borderWidth).grow()
+            add(borderTable).pad(borderWidth).width(width).minHeight(minHeight - totalPadding)
+        } else {
+            innerTable.background = ImageGetter.getBackground(outerColor)
+            add(innerTable).width(width).minHeight(minHeight - totalPadding)
         }
 
         touchable = Touchable.enabled
-        add(innerTable).width(width).minHeight(minHeight - totalPadding)
+        background = ImageGetter.getBackground(innerColor)
     }
-}
\ No newline at end of file
+}
+
+/*
+Layout if ruleset != null:
+
+       *Widgets*
+         Text colour                     Background Colour
+           Align                           Padding
+
++----- *NationTable* ----------------------------------------+
+|                                        getInnerColor       |
+| +---- *borderTable* -------------------------------------+ |
+| |                                      getOuterColor     | |
+| | +--- *innerTable* -----------------------------------+ | |
+| | | +-- *titleTable* --------------------------------+ | | |
+| | | |   getInnerColor                  getOuterColor | | | |
+| | | |     *nationIndicator*   *leaderDisplayLabel*   | | | |
+| | | |   center or left/wrap            0: all sides  | | | |
+| | | +------------------------------------------------+ | | |
+| | |     White                          Dark-blue       | | |
+| | |   MarkupRenderer.render(getCivilopediaTextLines)   | | |
+| | |     left/wrap                      10: all sides   | | |
+| | |                                                    | | |
+| | +----------------------------------------------------+ | |
+| +--------------------------------------------------------+ |
++------------------------------------------------------------+
```

Comment:
DAMN

Reply:
Oh, sorry, I needed that to understand better where which padding affects what and that stuff... Could have taken it out after. The Scene2D debug flags don't always help.